### DISCUSSION TITLE
InputPicker has no close function?

### DISCUSSION THREAD

**Discussion by PiousVenom** [rsuite/rsuite#2086] [Q&A]
Is there a way to collapse the InputPicker after opening it, without clicking outside the component, or selecting an item? Even in the documentation example, that doesn't seem to be an option.

**Comment by SevenOutman** [comment]
There should be `open` and `close` methods on the `ref` to `InputPicker`.

```jsx
const pickerRef = useRef();

<InputPicker ref={pickerRef} />
```
